webdriver-spec.html

```diff
@@ -2024,12 +2024,17 @@
 
  <li><p><a>Handle any user prompts</a> and return its value if it is an <a>error</a>.
 
+ <li><p>If the <code>url</code> property
+  is missing from the <var>parameters</var> argument
+  or it is not a string,
+  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
  <li><p>Let <var>url</var> be the result of
-  <a>getting a property</a> named "<code>url</code>"
-  from the <var>parameters</var> argument.
+  <a>getting a property</a> named <code>url</code>
+   from the <var>parameters</var> argument.
 
- <li><p>If <var>url</var> is undefined,
-  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+ <li><p>If <var>url</var> is a malformed URL,
```

Comment:
We should change this to error when it is not an [Absolute URL with fragement](https://url.spec.whatwg.org/#syntax-url-absolute-with-fragment).  At the moment there is no explanation of what a malformed URL is.